Feature/readme (#1)
3AP-AG/programming-dojo: README.md

# 👩‍💻3AP Days - Programming Dojo👨‍💻

## What will we do?

> On Friday afternoon we will improve our programming skills in a pair / extrem programming like setup.

Reviewer: ```suggestion
On Friday afternoon we will improve our programming skills in a pair / extreme programming like setup.
```

> We will work in setup of 4 - 5 people which will be defined Friday morning.

Reviewer 2: ```suggestion
We will work in teams of 4 - 5 people which will be defined Friday morning.
```

## What do we need from you

> - Please `checkout` this repository

Reviewer 2: Would it make sense to `fork` so that each team has its own repository? By `forking` you would prevent to be influenced by another teams solution.

Author: As discussed on Slack I would rather `branch` as you will have to regularly switch laptops

- Read through the rest of the `README`

## What is a programming dojo

- Please read through this [definition](http://codingdojo.org/WhatIsCodingDojo/)

> - Addtional check out this 📺 [Youtube Video](http://www.youtube.com/watch?v=gav9fLVkZQc)

Reviewer: ```suggestion
- Additionally check out this 📺 [Youtube Video](http://www.youtube.com/watch?v=gav9fLVkZQc)
```

> - Addtional check out this 📺 [Youtube Video](http://www.youtube.com/watch?v=gav9fLVkZQc)

Reviewer 2: ```suggestion
- Addtionally, check out this 📺 [Youtube Video](http://www.youtube.com/watch?v=gav9fLVkZQc)
```

### Kata

> Please through these 3 Katas. Your group will solve at least one of these Kata.

Reviewer: ```suggestion
Please read through these 3 Katas. Your group will solve at least one of these Kata.
```

1. http://codingdojo.org/kata/GameOfLife/
2. http://codingdojo.org/kata/GildedRose/
3. http://codingdojo.org/kata/TexasHoldEm/

## 🚨Important information

> - Each group can decided on their own which language they want to use, e.g. Java, Javascript, Typescript

Reviewer: ```suggestion
- Each group can decide on their own which language they want to use, e.g. Java, Javascript, Typescript
```

> - We don't recommend to use any libraries what so ever

Reviewer: I would allow use of testing libraries (e.g. JUnit, Spock)

Author: Good point

> - Use teh IDE you feel most comfortable in

Reviewer: ```suggestion
- Use the IDE you feel most comfortable in
```

> - To starte create a `branch` of your group

Reviewer 2: ```suggestion
- To start, create a `branch` for your group
```
